pyne/r2s.py

```diff
@@ -230,31 +230,111 @@ def total_photon_source_intensity(m, tag_name, sub_voxel=False):
             sv_data = ve_data[num_e_groups*svid:num_e_groups*(svid+1)]
             intensity += vol * np.sum(sv_data)
     return intensity
- 
+
 def tag_e_bounds(m, e_bounds, tag_name='e_bounds'):
-    """This function tags the energy boundaries to the PyMOAB mesh
-    instance as a sparse tag for the purpose of photon source sampling.
+    """This function tags the energy boundaries of photon source to the PyMOAB
```

Comment:
```suggestion
    """This function tags the energy boundaries of photon source to the PyNE
```

I missed this last time.